### PR TITLE
Add eslint, eslint_d linters

### PR DESCRIPTION
**Pull request by sirreal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@
 
 - [x] [clang-tidy](https://clang.llvm.org/extra/clang-tidy/)
 - [ ] [codespell](https://github.com/codespell-project/codespell)
+- [ ] [eslint_d](https://www.npmjs.com/package/eslint_d)
```
Comment (BBboy01) [inline README.md:39]:
```suggestion
- [ ] [eslint_d](https://github.com/mantoni/eslint_d.js)
```

github URL maybe better?

**Comment by xiaoshihou514** [comment]
@sirreal Could you kindly rebase? Or use the sync button via webui.

**Comment by sirreal** [comment]
Rebased 👍

**Comment by sirreal** [comment]
I added `find`, added this to the linter init, and added `eslint`. Eslint and eslint_d use nearly the same config, if there's a better way to share that config let me know.

**Pull request merged**